c/build_all/linux/setup.sh

```diff
@@ -12,7 +12,7 @@ repo_name_from_uri()
 }
 
 scriptdir=$(cd "$(dirname "$0")" && pwd)
-deps="curl libcurl4-openssl-dev uuid-dev uuid g++ make cmake git"
+deps="curl libcurl4-openssl-dev uuid-dev uuid g++ make cmake unzip openjdk-7-jre"
```

Comment:
I saw messages like that too, but they're just warnings:
-- Could NOT find Java (missing:  Java_JAVA_EXECUTABLE Java_JAR_EXECUTABLE Java_JAVAC_EXECUTABLE Java_JAVAH_EXECUTABLE Java_JAVADOC_EXECUTABLE)
-- Cannot find both Java and Maven: testing disabled for Proton-J

CMake successfully builds Proton-C and skips components that it can't build, like Proton-J.

That said, someone just told me they've seen this warning pop up *as a dialog* in their UI (e.g., when running this script in OSX).  That's annoying.  Let me see whether CMake has options for silencing that sort of thing.  Otherwise we'll take this change and worry about removing the JRE dependency later.